lib/ransack/helpers/form_helper.rb

```diff
@@ -94,7 +94,7 @@ def initialize(search, attribute, args, params)
           @current_dir    = existing_sort_direction
           @label_text     = extract_label_and_mutate_args!(args)
           @options        = extract_options_and_mutate_args!(args)
-          @hide_indicator = @options.delete :hide_indicator
+          @hide_indicator = indicator?
```

Comment:
The `indicator?` method is a bit misnamed, it is really checking if Ransack should `hide_indicator?` and the logic in that method appears to be backward.

It might be better to inline this method in the constructor with this, WDYT?

```ruby
@hide_indicator = @options.delete(:hide_indicator) || Ransack.options[:hide_sort_order_indicators]
```